lib/empty-example/sketch.js

```diff
@@ -1,7 +1,7 @@
 function setup() {
-  // put setup code here
-}
-
-function draw() {
-  // put drawing code here
-}
\ No newline at end of file
+    // put setup code here
+  }
+  
+  function draw() {
+    // put drawing code here
+  }
```

Comment:
Can the changes here be removed as it is not related to the issue?

Reply:
Actually, this is not really a change. I used the sketch.js file in the empty example to test my changes in the local machine. After I was done, I replaced it with the original template that it had.

Reply:
It is registered here as a change because the lines highlighted have additional white spaces at the beginning which shouldn't be there.